src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,84 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: React.FC<Props> = ({ person, people }) => {
+  const { sex, born, died, name, fatherName, motherName, slug } = person;
+  const { personSlug } = useParams();
+
+  const [searchParams] = useSearchParams();
+  const search = searchParams.toString();
+
+  const motherSlug = people.find(
+    ({ name: personName }) => personName === motherName,
+  );
+
+  const fatherSlug = people.find(
+    ({ name: personName }) => personName === fatherName,
+  );
```

Comment:
- You look for the person, not for the slug, naming is incorrect.
- You do the same in both cases, so you can create a function